### PR TITLE
comp: alert 

### PR DESCRIPTION
**Pull request by jsuryakt**
Please review alert component

image

### CODE REVIEW

**Review by arulk8 — APPROVED** [review]

File: Components/alert/alert.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>alert - minimal ui</title>
+    <link rel="stylesheet" href="/Components/install/install.css" />
+    <link rel="stylesheet" href="/prism.css" />
+    <link rel="stylesheet" href="/minimal.css" />
+    <link rel="stylesheet" href="alert.css" />
```
Comment (krgarima) [inline Components/alert/alert.html]: You can import all CSS files in a single CSS file.

File: minimal.css [hunk 2]
```diff
@@ -44,26 +50,48 @@ code {
...
+/* alert  */
+
+.alert {
+  display: flex;
+  justify-content: center;
+  gap: 0.5rem;
+  width: 25rem;
+  padding: 1rem;
+  border-radius: 0.3rem;
+  color: var(--white);
```
Comment (krgarima) [inline minimal.css]: You can write CSS variables as var(--white-color).

File: minimal.css [hunk 3]
```diff
@@ -44,26 +50,48 @@ code {
...
+}
+
+.primary {
+  background-color: var(--overlap-clr);
+}
+.error {
+  background-color: var(--error);
```
Comment (krgarima) [inline minimal.css]: Same here var(--error-color). It tells the reviewer var(--error-color) is a color variable.

**Review by krgarima — COMMENTED** [review]
Overall, the site looks good and clean.
Follow these suggestions when you have time:
1. You can add buttons. On clicking a button, alert pops up.
2. You can add some more variety of alerts.

**Comment by SuchitaGhadge** [comment]
It would look better if main content is at the center of the page. Just my suggestion. 
Looks good.
<details><summary>Browser metadata</summary>





```
Path:      /components/alert/alert.html
Browser:   Chrome 98.0.4758.102 on Windows 10
Viewport:  1536 x 746 @1.25x
Language:  en-GB
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-5--minimal-ui-library.netlify.app%2Fcomponents%2Falert%2Falert.html&os=Windows&os_version=10&browser=Chrome&browser_version=98.0&resolution=1536x746&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-5--minimal-ui-library.netlify.app/components/alert/alert.html) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=62137fc3228c1c0008beed64&commentID=62138a0ee5e2f10a8a13df78&resolution=resolved)

**Comment by SuchitaGhadge** [comment]
Overall looks good. 
If you can align the main content to the center then it would look good in my opinion.

File: minimal.css (hunk 2 above)
Comment (jsuryakt) [reply minimal.css]: Okay thanks

File: Components/alert/alert.html (hunk 1 above)
Comment (jsuryakt) [reply Components/alert/alert.html]: Thanks
Implemented it

File: minimal.css (hunk 2 above)
Comment (jsuryakt) [reply minimal.css]: Implemented

File: minimal.css (hunk 3 above)
Comment (jsuryakt) [reply minimal.css]: Implemented

**Pull request merged**